pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-798_drop_unique_index.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet author="pegah" id="PEPPER-798_drop_indices">
```

Comment:
It's not clear to me why we have to drop the indexes here in order to have proper unique checks at the code level.  Dropping the indexes will have performance ramifications, and if we need to do any direct db work, the lack of the unique indexes here will increase the risk of dirty data.  Can we take another look at keeping the indexes (or changing to unique constraints) and enabling code-level unique checks?